examples/storm-perf/src/main/java/org/apache/storm/perf/JCQueuePerfTest.java

```diff
@@ -28,15 +27,15 @@
     // Usage: Let it and then explicitly terminate.
     // Metrics will be printed when application is terminated.
     public static void main(String[] args) throws Exception {
-//        oneProducer1Consumer(1000);  // -- measurement 1
-//        twoProducer1Consumer(1000);    // -- measurement 2
-//        threeProducer1Consumer(1);   // -- measurement 3
+        //        oneProducer1Consumer(1000);  // -- measurement 1
+        //        twoProducer1Consumer(1000);    // -- measurement 2
+        //        threeProducer1Consumer(1);   // -- measurement 3
```

Comment:
Do we want to just remove these? Otherwise,  delete the spaces between `//` and the actual code, e.g. `//threeProducer1Consumer(1);` to make it look better